lib/openstack/amqp/openstack_qpid_event_monitor.rb

```diff
@@ -20,9 +24,9 @@ def self.test_connection(options = {})
       connection.open
       connection.open?
     rescue => e
-      log_prefix = "MIQ(#{self.name}.#{__method__}) Failed testing qpid amqp connection for #{options[:hostname]}. "
-      $log.info("#{log_prefix} The Openstack AMQP service may be using a different provider.  Enable debug logging to see connection exception.") if $log
-      $log.debug("#{log_prefix} Exception: #{e}") if $log
+      log_prefix = "MIQ(#{self.name}.#{__method__}) Failed testing QPid AMQP connection for #{options[:hostname]}. "
+      $log.info("#{log_prefix} The Openstack AMQP service may be using a different provider.") if $log
+      $log.info("#{log_prefix} Exception: #{e}") if $log
```

Comment:
It's tough .. the problem is that these occur during the basic process of determining if this is the right amqp client.  If this fails (and is logged), it's quite possible that another amqp client will still work.

Logging this as an error will start to raise flags and users will react to the error instead of reading the message.

That was my thought on keeping it info.  We could warn, I suppose.